### PR TITLE
add keras/text_classification_with_hub.ipynb indonesian translation

### PR DESCRIPTION
**Pull request by csahat**

### CODE REVIEW

**Comment by febriaroo** [comment]
LGTM

**Comment by lamberta** [comment]
CI is failing: Please run notebooks through [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py). This will make diffs easier to read.

File: site/id/tutorials/keras/text_classification_with_hub.ipynb [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+        "colab_type": "text",
+        "id": "hKY4XMc9o8iB"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/keras/text_classification_with_hub\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />Lihat di TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Jalankan di Google Colab</a>\n",
```
Comment (lamberta) [inline site/id/tutorials/keras/text_classification_with_hub.ipynb:93]:
```suggestion
        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/id/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Jalankan di Google Colab</a>\n",
```

File: site/id/tutorials/keras/text_classification_with_hub.ipynb [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />Lihat sumber kode di GitHub</a>\n",
```
Comment (lamberta) [inline site/id/tutorials/keras/text_classification_with_hub.ipynb:96]:
```suggestion
        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/id/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />Lihat sumber kode di GitHub</a>\n",
```

File: site/id/tutorials/keras/text_classification_with_hub.ipynb [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Unduh notebook</a>\n",
```
Comment (lamberta) [inline site/id/tutorials/keras/text_classification_with_hub.ipynb:99]:
```suggestion
        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs-l10n/site/id/tutorials/keras/text_classification_with_hub.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Unduh notebook</a>\n",
```

**Pull request merged**